cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-databases-drop-database, DROP DATABASE ... [IF EXISTS]>>                                                                                                         | Database administration       | Deletes a specified database.
+| <<administration-databases-start-database, START DATABASE>>                                                                                                                       | Database administration       | Starts up a specified database.
+| <<administration-databases-stop-database, STOP DATABASE>>                                                                                                                         | Database administration       | Stops a specified database.
+| <<administration-security-roles-show, SHOW [ALL \| POPULATED] ROLES [WITH USERS]>>                                                                                                | User and role administration  | Returns information about all or populated roles, optionally including the assigned users.
+| <<administration-security-roles-create, CREATE [OR REPLACE] ROLE ... [IF NOT EXISTS] ... [AS COPY OF]>>                                                                           | User and role administration  | Creates new roles.
+| <<administration-security-roles-drop, DROP ROLE [IF EXISTS]>>                                                                                                                     | User and role administration  | Deletes a specified role.
+| <<administration-security-users-show, SHOW USERS>>                                                                                                                                | User and role administration  | Returns information about all users.
+| <<administration-security-users-create, CREATE [OR REPLACE] USER ... [IF NOT EXISTS] SET PASSWORD ... [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>   | User and role administration  | Creates a new user and sets the password for the new account. Optionally the account status can also be set and if the user should change the password upon first login.
+| <<administration-security-users-drop, DROP USER ... [IF EXISTS]>>                                                                                                                 | User and role administration  | Deletes a specified user.
+| <<administration-security-users-alter, ALTER USER ... [SET PASSWORD ...] [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>                                | User and role administration  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
```

Comment:
I also found a small thing missing in the section about modifying a user (https://neo4j.com/docs/cypher-manual/4.0-preview/administration/security/users-and-roles/#administration-security-users-alter). We should mention something there about all other values being unaffected if not specified (for example if you have `password change required` and change only the password `ALTER USER name SET PASSWORD 'secret'` then you still have `password change required`)

Reply:
@craigtaverner said he might fix this when building and reviewing the pr

Reply:
I leave this to Craig then